dali/operators/sequence/optical_flow/optical_flow.h

```diff
@@ -78,7 +78,11 @@ class OpticalFlow : public StatelessOperator<Backend> {
 #endif
   }
 
-  ~OpticalFlow();
+  ~OpticalFlow() {
+#if NVML_ENABLED
+    nvml::Shutdown();
```

Comment:
Isn't that a problem if we have a couple of the ops instances (two pipelines for instance) and one of them goes out of scope while the other remains in use?

Reply:
I think it should not (based on [the docs](https://docs.nvidia.com/deploy/nvml-api/group__nvmlInitializationAndCleanup.html#group__nvmlInitializationAndCleanup_1gcb6e78434b4d14250ce79c3e7494f06d))
```
This method, should be called once before invoking any other methods in the library. A reference count of the number of initializations is maintained. Shutdown only occurs when the reference count reaches zero.
```